### ISSUE TITLE
Irradiance Model Results

### ISSUE THREAD

**Issue by jacobbieker** [openclimatefix/pv-pseudo-experiments#5]
Here are some overviews of the results from Pseudo-irradiance model training

## Detailed Description
The models are trained with batches of 256, and 256km areas around random PV sites. 

# Current Issues
The loss drops dramatically quickly then stays flat, regardless of batch norm, number of examples, channels, etc.

Current (May 2nd) best model is located [here](https://wandb.ai/openclimatefix/PvIrradiance/runs/a9lrrco1)

**Comment by jacobbieker** [comment]
Plots from May 2nd model Validation for batch 1:
![media_images_Input_val_1_88332_f156ddcb678f4181b001](https://user-images.githubusercontent.com/7170359/235620975-eb75ec5b-e262-42ab-96e2-a80c94321781.png)
![media_images_GT_val_1_88333_910f3703576293cde45f](https://user-images.githubusercontent.com/7170359/235621007-cbb1d790-8005-4a4a-8c46-22cc893f122c.png)
![media_images_Pred_val_1_88334_49fa69ab098b64ee5b65](https://user-images.githubusercontent.com/7170359/235621044-8e9921b4-1e8a-4ece-9b7d-e9be60965bb4.png)
